tremor-cli/Cargo.toml

```diff
@@ -39,13 +39,7 @@ serde_yaml = "0.8"
 signal-hook = "0.3"
 signal-hook-async-std = "0.2"
 simd-json = { version = "0.5", features = ["known-key"] }
-# we need to stick with 0.2.26 as it includes its own libc
-# which allows us to build on older systems like centos 7
-# issues to track until we can loosen those restrictions:
-#   - https://github.com/microsoft/snmalloc/issues/328
-#   - https://github.com/SchrodingerZhu/snmalloc-rs/issues/145
-snmalloc-rs = { version = "=0.3.1", optional = false }
-snmalloc-sys = { version = "=0.3.1", optional = false }
+snmalloc-rs = { version = "0.2" }
```

Comment:
sneaky little hobbitses. Can you add a comment on why the downgrade is necessary?